Manifests/ManagedPreferencesApplications/com.secondsonconsulting.baseline.plist

```diff
@@ -431,6 +431,82 @@ A profile can consist of payloads with different version numbers. For example, c
 			<key>pfm_type</key>
 			<string>array</string>
 		</dict>
+		<dict>
+			<key>pfm_app_min</key>
+			<string>2.2</string>
+			<key>pfm_description</key>
+			<string>Define files which you want Baseline to wait for. Use this for items not directly installed by Baseline, like VPP or MDM installed apps.</string>
+			<key>pfm_name</key>
+			<string>WaitFor</string>
+			<key>pfm_subkeys</key>
+			<array>
+				<dict>
+					<key>pfm_subkeys</key>
+					<array>
+						<dict>
+							<key>pfm_description</key>
+							<string>The name you want to appear in the SwiftDialog menu for this item we are waiting for.</string>
+							<key>pfm_name</key>
+							<string>DisplayName</string>
+							<key>pfm_title</key>
+							<string>Display Name</string>
+							<key>pfm_type</key>
+							<string>string</string>
+						</dict>
+						<dict>
+							<key>pfm_description</key>
+							<string>A path to the file you want Baseline to wait for.</string>
+							<key>pfm_name</key>
+							<string>Path</string>
+							<key>pfm_title</key>
+							<string>Path</string>
+							<key>pfm_type</key>
+							<string>string</string>
+						</dict>
+						<dict>
+							<key>pfm_description</key>
+							<string>The icon you want to appear in the SwiftDialog menu for this item. Can be a URL or local file path.</string>
+							<key>pfm_name</key>
+							<string>Icon</string>
+							<key>pfm_title</key>
+							<string>Icon</string>
+							<key>pfm_type</key>
+							<string>string</string>
+						</dict>
+						<dict>
+							<key>pfm_app_min</key>
+							<string>2.1</string>
```

Comment:
Likely sneaked in by mistake, seeing that the container is 2.2 minimum.

Reply:
That 2.1 is actually on purpose, back filling something I missed with my last update.

Reply:
If that's the case then the container, `WaitFor`, was already supported in 2.1 and therefore should declare it as minimum instead of 2.2. Then assuming that `DisplayName`, `Path`, and `Icon` are new, they could declare their minimum as 2.2.

Although that's how it should be marked up if I understood you correctly, I suspect there's still some detail missing because if I try to understand the role of `WaitFor`, it doesn't really make sense to me that it was introduced in 2.1 with only `Subtitle`.

Reply:
Apologies for the confusion, you are correct in your original assessment that I had the wrong pfm_app_min for the `WaitFor` . I misread which key we were discussing. I've fixed this, thank you as always for your attention to detail.